pkg/aws/service.go

```diff
@@ -197,8 +208,9 @@ func FromEnv(ctx context.Context) Config {
 					MinVersion: tls.VersionTLS12,
 				},
 			},
-			IndexerURL:             mustGetEnv("IPNI_ENDPOINT"),
-			PublisherAnnounceAddrs: []string{ipniPublisherAnnounceAddress},
+			IndexerURL:                  mustGetEnv("IPNI_ENDPOINT"),
+			PublisherAnnounceAddrs:      []string{ipniPublisherAnnounceAddress},
+			PublisherDirectAnnounceURLs: ipniPublisherDirectAnnounceURLs,
```

Comment:
would you mind renaming these fields so that it is clear what they are? Probably prefixing them with `IPNI` would be enough.